pkg/glusterfs/controllerserver.go

```diff
@@ -29,6 +29,44 @@ type ControllerServer struct {
 	*GfDriver
 }
 
+// csiDrvParam stores csi driver specific request parameters.
+// This struct will be used to gather specific fields of CSI driver (csiDrvName, csiDrvVersion..etc)
+// and also gather parameters passed from SC and not part of gd2volcreate api.
+// glusterCluster - The resturl of gluster cluster
+// glusterUser - The gluster username who got access to the APIs.
+// glusterVersion - Says the version of the glustercluster running in glusterCluster endpoint.
+// compMatrix - map which will be internally defined by the driver to make a compatibility
+//              version matrix between CSI driver and gluster cluster. All these fields are optional and can be used if needed.
+
+type csiDrvParam struct {
+	glusterCluster string
+	glusterUser    string
```

Comment:
@humblecI as discussed to make more sense we need to add the userToken field to the struct.
as we require both username and Token to connect to rest endpoints

Reply:
@Madhu-1 awesome, adding that too.